packaging_automation/templates/docker/postgres-13/postgres-13.tmpl.dockerfile

```diff
@@ -20,7 +19,7 @@ RUN apt-get update \
        ca-certificates \
        curl \
     && curl -s https://install.citusdata.com/community/deb.sh | bash \
-    && apt-get install -y postgresql-$PG_MAJOR-citus-{{project_minor_version}}=$CITUS_VERSION \
+    && apt-get install -y postgresql-$PG_MAJOR-citus-{{project_minor_version}}.=$CITUS_VERSION \
```

Comment:
Why do we have the extra `.` character here?

Reply:
And more importantly, how is it possible that the tests passed even though this `apt-install` line seems to be broken?

Reply:
Both with dot and without dot works. I added it just to make it same with other files. I removed all of the dots now in all applicable template files